pkg/api/types/formintf.go

```diff
@@ -0,0 +1,56 @@
+package types
```

Comment:
I think it'd be clearer to name this file `forminterface.go` or `form_interface.go`. It's easier to know what it contains rather than having to work out what `intf` refers to.

Reply:
sounds good